playwright.config.js

```diff
@@ -5,7 +5,7 @@ import { devices } from '@playwright/test';
 /** @type {import('@playwright/test').PlaywrightTestConfig<{ domain: string, showAllConsole: boolean }>} */
 const config = {
   forbidOnly: !!process.env.CI,
-  retries: process.env.CI ? 2 : 0,
+  retries: process.env.CI ? 0 : 0,
```

Comment:
Not sure, is this correct? `retries: process.env.CI ? 0 : 0,`

Maybe can simplify